stablehlo/dialect/AssemblyFormat.cpp

```diff
@@ -276,6 +278,73 @@ ParseResult parseDenseI64Array(OpAsmParser& parser,
   return success();
 }
 
+void printSliceRanges(OpAsmPrinter& p, Operation* op,
+                      DenseIntElementsAttr start_indices,
+                      DenseIntElementsAttr limit_indices,
+                      DenseIntElementsAttr strides) {
+  p << "[";
+  // Let's be safe if we're printing invalid IR somehow: this can't be parsed
+  // back!
+  if (start_indices.size() != limit_indices.size() ||
+      start_indices.size() != strides.size()) {
+    p << "start_indices: ";
+    llvm::interleaveComma(start_indices, p);
+    p << ", limit_indices: ";
+    llvm::interleaveComma(limit_indices, p);
+    p << ", strides: ";
+    llvm::interleaveComma(strides, p);
+    p << "]";
+    return;
+  }
+
+  llvm::interleaveComma(
+      llvm::zip(start_indices, limit_indices, strides), p,
+      [&](std::tuple<llvm::APInt, llvm::APInt, llvm::APInt> pack) {
+        auto [start, limit, stride] = pack;
+        p << std::get<0>(pack) << ":" << std::get<1>(pack);
```

Comment:
You probably meant to use `start` and `limit` here, instead of `std::get<0>` and `std::get<1>`?

Reply:
Indeed